src/SlashCommands.tsx

```diff
@@ -815,6 +815,28 @@ export const Commands = [
                         }),
                     );
                 }
+                const roomMatches = args.match(/^([!][^:]+:\S+)$/);
+                if (roomMatches) {
```

Comment:
But you can't write slash commands in a room in which you are holding an invite, you'd need to join the room to the ignore invites and then leave? That doesn't sound safe.

Reply:
Well, user feedback suggests that this is actually a common behavior:

1. Get tricked into a room.
2. Realize that you have been tricked.
3. Decide never to be invited in this room again.
4. ...but we don't have tools for that yet.

We definitely want to add the ability to ignore invites before joining the room, but we're planning to work on that as a later stage, as this will require deeper changes in the UI.

Reply:
But if you leave before you realise "oh maybe I should ignore future invites" then it'll be too late, you'll have to wait for a second invite, accept it, then finally you can ignore it.

Reply:
Absolutely.

Right now, we're trying to combine two things:
- handle at least one common case with minimal UI;
- lay the foundations for handling the other cases once we have a proper UI design (which we're working on in the background, but with no ETA as of yet).